robottelo/cli/discoveredhost.py

```diff
@@ -31,10 +31,28 @@ class DiscoveredHost(Base):
     def provision(cls, options=None):
         """Manually provision discovered host"""
         cls.command_sub = 'provision'
-        return cls.execute(cls._construct_command(options))
+        return cls.execute(cls._construct_command(options), output_format='csv')
 
     @classmethod
     def facts(cls, options=None):
         """Get all the facts associated with discovered host"""
         cls.command_sub = 'facts'
-        return cls.execute(cls._construct_command(options))
+        return cls.execute(cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def auto_provision(cls, options=None):
+        """Auto provision discovered host"""
+        cls.command_sub = 'auto-provision'
```

Comment:
I don't see any scenario covering auto-provision, are you planning to add a separate test for it?

Reply:
Yeah I was just adding all the relevant support for the discovery we can surely have some tests as well

Reply:
so are you considering same PR or new?

Reply:
Not in radar for this PR.